src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/SqlParameter.cs

```diff
@@ -984,9 +995,11 @@ private void CloneHelper(SqlParameter destination)
 
             destination._metaType = _metaType;
             destination._collation = _collation;
-            destination._xmlSchemaCollectionDatabase = _xmlSchemaCollectionDatabase;
-            destination._xmlSchemaCollectionOwningSchema = _xmlSchemaCollectionOwningSchema;
-            destination._xmlSchemaCollectionName = _xmlSchemaCollectionName;
+            if (_xmlSchemaCollection != null)
```

Comment:
If I am not mistaken, with the C# language updated to 9 in the driver recently I think we can use "is not null"

Reply:
Yes. It's a question of what style we want to use really. I like is not null but I've been trying to synchronise the style throughout the library with these merge PR's so if we want to do a !=null -> is not null change I think I'd want to do it in a separate PR over most of the files at once.

Reply:
NIT: The `CopyFrom()` function internally skips a null parameter and you can get rid of this line.

Reply:
If we skip the null check we'll create the instance for the current class even though the one we're copying from doesn't have one. We're trying to avoid creating them unless we have to.

Reply:
It's reasonable.

Reply:
For a path that's so little used I'm not too concerned about an extra null check. using xml schema is very rare.